src/apiserver/json_helper.cc

```diff
@@ -200,6 +207,137 @@ JsonReader& JsonReader::operator&(std::string& s) {  // NOLINT
     return *this;
 }
 
+JsonReader& JsonReader::operator&(std::shared_ptr<openmldb::sdk::SQLRequestRow>& parameter) {  // NOLINT
```

Comment:
json_helper is the util, do not do impl read/write methods in it. Move it to api_server_impl, or create a new file for all req&resp class.